mysql-test/include/analyze-format.inc

```diff
@@ -0,0 +1,3 @@
+# The time on ANALYSE FORMAT=JSON is rather variable
+
+--replace_regex /("(r_total_time_ms|r_buffer_size)": )[^, \n]*/\1"REPLACED"/
```

Comment:
The field names are intact as indicated by the result files. The outer brackets form a capture and \1 places the capture in the result and only replaces the value of the parameter.